src/mdx/_Code.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Highlight, { defaultProps as highlightProps } from 'prism-react-renderer';
+import classNames from 'classnames/bind';
+import styles from './MarkdownTags.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Children for the component.
+   */
+  children: PropTypes.node,
+  /**
+   * ClassName for the component.
+   */
+  className: PropTypes.string,
+
+};
+
+const Code = ({ children, className }) => {
+  const language = (className || '').replace(/language-/, '');
+  return (
+    // We remove the theme to just use the css theme.
+    <Highlight {...highlightProps} code={children} language={language} theme={undefined}>
```

Comment:
Are these default props from `prism-react-renderer` not defined on the Highlight component? Or why are you spreading them on it?